plugins/dataformat-csv/build.gradle

```diff
@@ -0,0 +1,112 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+apply plugin: 'opensearch.opensearchplugin'
+
+opensearchplugin {
+  name = 'dataformat-csv'
+  description = 'CSV data format plugin for OpenSearch DataFusion'
+  classname = 'org.opensearch.datafusion.csv.CsvDataFormatPlugin'
```

Comment:
We want to decouple the package namespacing?

Reply:
Yes , i'll work with mohit , this whole dataformat plugin will probably be moved/designed as something that can work for both query and indexing. I have todos in plugin class for the same.